### DISCUSSION TITLE
Does any one know if it is possible to change movement names in Sheltermanager?

### DISCUSSION THREAD

**Discussion by ADIN2014** [sheltermanager/asm3#1461] [Q&A / Help] [answered]
We have ceased using the terminology of Trial Adoption within our rescue as we find that people sometimes view it as something to give up on.  We now use Supported Adoption Period.  We would like to change the terminology on Sheltermanager - but sure if it is editable.  Does anyone know?

**Comment by Tetzor** [comment, marked as answer]
I'm afraid movement types are one of the areas of the system that can't be customised unfortunately.

**Reply by ADIN2014** [reply]
Ok thank you.
